config/templates/hiveconfig.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: hive-config
+
+parameters:
+  - name: HIVE_NS
+    displayName: Hive Namespace
+    description: Namespace where core hive components will run.
+    value: hive
+
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: HiveConfig
+    metadata:
+      name: hive
+    spec:
+      logLevel: debug
```

Comment:
Do we really want the default here to be debug?

Reply:
For make deploy I could go either way, I'll flip to info.

Reply:
Would it make sense to make it configurable and use the `RUN_LOG_LEVEL` variable in the make deploy call?

Reply:
Sounds handy, will add.